scene/resources/audio_stream_polyphonic.cpp

```diff
@@ -66,11 +74,62 @@ void AudioStreamPolyphonic::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "polyphony", PROPERTY_HINT_RANGE, "1,128,1"), "set_polyphony", "get_polyphony");
 }
 
-AudioStreamPolyphonic::AudioStreamPolyphonic() {
+AudioStreamPolyphonic::AudioStreamPolyphonic() {}
+AudioStreamPolyphonic::~AudioStreamPolyphonic() {
+	playbacks.clear();
```

Comment:
```suggestion
```

This is not necessary. `playbacks` will be destructed (cleared) automatically by C++ when `AudioStreamPolyphonic` destructs.

You could move both definitions to the `.h` file since they're empty.